pages/resources/guild.mdx

```diff
@@ -1399,6 +1403,99 @@ This endpoint is not usable by user accounts.
 | query  | string  | Query to match username(s) and nickname(s) against  |
 | limit? | integer | Max number of members to return (1-1000, default 1) |
 
+<RouteHeader method="POST" url="/guilds/{guild.id}/members-search">
+  Search Guild Members
+</RouteHeader>
+
```

Comment:
As this is a rename, would it be worth it to have "Did you mean: Query Guild Members" since the endpoints are basically identical and old doc urls will point to the new one

Reply:
They're right next to each other so I think it's fine